Sources/SwiftProtobuf/Google_Protobuf_Duration+Extensions.swift

```diff
@@ -169,6 +169,46 @@ extension Google_Protobuf_Duration {
     }
 }
 
+@available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *)
+extension Google_Protobuf_Duration {
+    /// Creates a new `Google_Protobuf_Duration` by rounding a `Duration` to
+    /// the nearest nanosecond according to the given rounding rule.
+    ///
+    /// - Parameters:
+    ///   - duration: The `Duration`.
+    ///   - rule: The rounding rule to use.
+    public init(
+        rounding duration: Duration,
+        rule: FloatingPointRoundingRule = .toNearestOrAwayFromZero
```

Comment:
`.toNearestOrEven` should be the default.  (This is the default rounding rule for IEEE 754, for instance, and should be preferred in almost all cases.)

Reply:
I chose this to align with the no-argument form of `BinaryFloatingPoint.round()` and `.rounded()` in the standard library: https://github.com/swiftlang/swift/blob/8dd7669816faf5228e2e4723988bef6f8b982662/stdlib/public/core/FloatingPoint.swift#L1659-L1667

Since this is the default for the standard library's behavior, I'd be worried about making this behave subtly differently. Do you still feel it should be changed?

Reply:
Leave it as-is then, I guess.